README.md

```diff
@@ -129,6 +131,10 @@ Si deseas agregar algún canal o tienes alguna sugerencia, puedes usar [@openve]
 * [Noticias](https://t.me/ResumenNoticiero) - @Carrasquel
 * [Paranormales](https://t.me/paranormales) - @Richzendy - @Willicab
 
+### Bots
```

Comment:
![all your base are belong to us](https://media.giphy.com/media/jnBXe38Ww2XAI/giphy.gif)